src/assests/styles/auth.css

```diff
@@ -0,0 +1,63 @@
+@import "./colors.css";
+
+.login, .signup{
+     margin:3rem auto;
+     padding: 1rem;
+     width: max-content;
+     box-shadow: var(--all-side-shadow);
+}
+
+.heading{
+     font-size: 1.5rem;
+     margin: auto;
+}
+
+.login-form{
+     padding: 10px 1rem;
+}
+
+.input-block{
+     margin:10px;
+     align-items: unset;
+     padding: 0;
+}
+
+.input-block label{
+     font-size: 1.2rem;
+     margin-bottom:10px;
+}
+
+.input-block input{
+     padding:0.5rem 1rem;
+}
+
+.input-block .btn-link{
+     text-decoration: none;
+     color: var(--text-col);
+     font-style: italic;
+}
+
+.btn-link-login{
+     font-weight: bold;
+     font-size: 1.4rem;
+}
+
+.login-to{
+     display: flex;
+     flex-direction: row;
+     align-items: center;
+}
+
+.input-block-name{
+     display: flex;
+}
+
+.input-block-check{
+     font-size: 1.2rem;
+     display: flex;
```

Comment:
Could make a utility class for repetitive CSS properties like flex.

Reply:
ok